packages/ui/src/components/sidebar/sidebar-item.tsx

```diff
@@ -181,19 +182,19 @@ const SidebarItemTrigger = forwardRef<HTMLButtonElement | HTMLAnchorElement, Sid
           'cn-sidebar-item-content-complete': withDescription && withRightElement
         })}
       >
-        {withIcon && props.icon && (
-          <>
-            {withDescription && (
-              <div className="cn-sidebar-item-content-icon cn-sidebar-item-content-icon-w-border">
-                <IconV2 name={props.icon} size="md" />
-              </div>
-            )}
-            {!withDescription && <IconV2 name={props.icon} size="sm" className="cn-sidebar-item-content-icon" />}
-          </>
-        )}
+        {(withIcon || withFallback) &&
+          (withDescription ? (
+            <div className="cn-sidebar-item-content-icon cn-sidebar-item-content-icon-w-border">
+              <IconV2 name={props.icon} size="md" fallback="stop" />
+            </div>
+          ) : (
+            <IconV2 name={props.icon} size="sm" fallback="stop" className="cn-sidebar-item-content-icon" />
```

Comment:
Above we have used `cn-sidebar-item-content-icon` class to a `div`. Here we are applying to `IconV2`. Is it intentional? What is the purpose of this class?

Reply:
Absolutely. It's applied to each image component and used for proper positioning in CSS 

<img width="776" height="376" alt="image" src="https://github.com/user-attachments/assets/e91293f0-d377-446d-825f-f11981357f8c" />